File: Python/(1) Multiples of 3 and 5.py
answer = 0

"""
'answer' variable is the sum of
all the multiples of 3 or 5 below 1000
"""

for i in range(1000):  # cycle for find the sum
    if i % 3 == 0 or i % 5 == 0:
        answer += i    # '+=' same as 'answer = answer + i'
print(answer)

# https://projecteuler.net/problem=1 => problem description

File: C++/C++/(1) Multiples of 3 and 5.cpp
#include <iostream>
using namespace std; 

int main()
{
    int answer = 0; 
    int o = 10; 
    for (int i = 0;  i < 1000; i++) {
        if ((i % 3 == 0) || (i % 5 == 0)) {
            answer = answer + i; 
        }
    }
    cout << "Answer is: " << answer << endl;
}
